valhalla/baldr/graphconstants.h

```diff
@@ -358,7 +358,8 @@ inline std::string to_string(Use u) {
   return i->second;
 }
 
-enum class TaggedName : uint8_t { // must start at 1 due to nulls
+enum class TaggedValue : uint8_t { // must start at 1 due to nulls
+  kZLevel = 1,
```

Comment:
Why is this kZLevel = 1?  Shouldn't it be kZLevel = static_cast<uint8_t>('3'),

Reply:
@SiarheiFedartsou @kevinkreiser ^

Reply:
@gknisely no, it should be the integer `1`, just like bridge and tunnel should have been integers in the first place. this was the bug fix we recently did. remember that strings are just collections of bytes they dont have to be readible characters. this allows us the possibility to keep lots of different tag/value combinations. this change aligns perfectly with the fix in #3264 for more explanation please read the issue i wrote up and the comment i made on the original tunnel pr

Reply:
@kevinkreiser got it.  i missed that we were writing out as strings